### PR TITLE
Reduce README

### PR DESCRIPTION
**Pull request by GBLin5566**
Model building example is in the tutorial now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,75 +12,7 @@ pip install yoctol-nlu
 
 ## Usage
 
-### Intent Classifier Service
-
-#### Fetch Model & One Line Prediction
```
Comment (stegben) [inline README.md]: I think it's OK to contain this session since it's the core functionality.

Comment (GBLin5566) [reply README.md]: ok, gonna put it back

**Comment by SoluMilken** [comment]
Should we have a version for Entity???

**Comment by GBLin5566** [comment]
I'm working on it.

**Comment by SoluMilken** [comment]
You can obtain tools of entity preprocessing from `evaluation/utils.py`.

**Pull request merged**